template/runtime/src/lib.rs

```diff
@@ -475,6 +476,13 @@ impl_runtime_apis! {
 			}
 		}
 
+		fn storage_at(address: H160, index: U256) -> H256 {
+			let index = H256::from_slice(
```

Comment:
I don't think this is the actual encoding. It should simply be `U256` encoded in big-endian format. So something like `H256::from(index.to_big_endian())`.